src/components/diagrams/singleLineDiagram/single-line-diagram.js

```diff
@@ -1011,26 +1012,36 @@ const SizedSingleLineDiagram = forwardRef((props, ref) => {
                     <Box flexGrow={1}>
                         <Typography>{props.diagramTitle}</Typography>
                     </Box>
-                    <IconButton
-                        className={classes.actionIcon}
-                        onClick={minimizeSld}
-                    >
-                        <MinimizeIcon />
-                    </IconButton>
-                    <IconButton
-                        className={
-                            pinned ? classes.actionIcon : classes.pinRotate
-                        }
-                        onClick={pinSld}
-                    >
-                        {pinned ? <PushPinIcon /> : <PushPinOutlinedIcon />}
-                    </IconButton>
-                    <IconButton
-                        className={classes.close}
-                        onClick={onCloseHandler}
-                    >
-                        <CloseIcon />
-                    </IconButton>
+                    <Box>
```

Comment:
We can remove this box nope ?

Reply:
Nope, if we want to keep the icons stuck to the top and have the label to uses more than one line of text, we need the two levels, outer for vertical concern and inner for horizontal alignment.

Reply:
Ah yes, you're right